README.md

```diff
@@ -109,29 +109,27 @@ efficient format. Input files should be sorted by ascending time.
 
 |Name|Unit|Type|Description|
 |---|---|---|---|
-| obs_id | None | str |Unique observation ID for the observation |
+| obs_id | None | str | Unique observation ID for the observation |
 | exposure_id | None | str | Exposure or Image ID from which observation was measured |
-| mjd | days | float | MJD of the observation in UTC** |
+| mjd | days | float | MJD of the observation in UTC[^1] |
 | ra  | degree | float | Right Ascension (J2000) |
 | dec  | degree | float | Declination (J2000) |
-| ra_sigma  | degree | float | 1-sigma uncertainty in Right Ascension |
-| dec_sigma  | degree | float | 1-sigma uncertainty in Declination |
+| ra_sigma  | degree | float | 1-sigma uncertainty in Right Ascension (Nullable)[^2] |
+| dec_sigma  | degree | float | 1-sigma uncertainty in Declination (Nullable)[^2] |
 | mag  | None | float | Photometric magnitude measured for observation |
-| mag_sigma  | None | float | 1-sigma uncertainty in photometric magnitude |
+| mag_sigma  | None | float | 1-sigma uncertainty in photometric magnitude (Nullable)[^2]|
 | filter | None | str | Filter/bandpass in which the observation was made |
 | exposure_mjd_start | days | float | Start MJD of the exposure in UTC |
 | exposure_mjd_mid | days | float | Midpoint MJD of the exposure in UTC |
 | exposure_duration | seconds | float | The length of the exposure |
 | observatory_code | None | str | MPC observatory code for the observatory/observing program |
 
-Note:  
-** The time at which the observation is reported may be different than the exposure midpoint time to account for effects such as shutter motion
 
 ## Precovery Results
 
 |Name|Unit|Type|Description|
 |---|---|---|---|
-| mjd | days | float | MJD of the observation in UTC** |
+| mjd | days | float | MJD of the observation in UTC[^1] |
 | ra_deg  | degree | float | Right Ascension (J2000) |
 | dec_deg  | degree | float | Declination (J2000) |
 | ra_sigma_arcsec  | arcsecond | float | 1-sigma uncertainty in Right Ascension |
```

Comment:
This is really the area that's more important for documentation: Precovery's *outputs* might be NaNs for certain fields.

Reply:
I'll update the description and describe frame candidates as well.